src/runtime/virtcontainers/clh_test.go

```diff
@@ -373,6 +394,9 @@ func TestCloudHypervisorNetRateLimiter(t *testing.T) {
 func TestCloudHypervisorBootVM(t *testing.T) {
 	clh := &cloudHypervisor{}
 	clh.APIClient = &clhClientMock{}
+
+	vmAddNetPutRequest = func(clh *cloudHypervisor) error { return nil }
```

Comment:
Since `vmAddNetPutRequest` is a global, I think you might be missing calls to save the original function and a `defer` to reset it when this function ends?